guidelines/REUSE.md

```diff
@@ -133,8 +209,20 @@ _Assuming file https://my.company.com/definitions/models.json_
 ```
 
 
+### Definitions
+
+Reuse schema definitions by creating a repository of definitions.  This is done by simply hosting a file or set of files for commonly used definitions across a company or organization.
+
+Refer to [External references](#external-references) for referencing strategies.
```

Comment:
<again, existing wording I tried to preserve>

You could say that `parameters`, `responses`, `definitions` are "repositories of definitions", which happen to reside in the same document (vs external files).